docs/changes/3.0.0.rst

```diff
@@ -269,7 +269,7 @@ Fixed
   could raise an unexpected ``AttributeError`` when running under certain
   applications servers such as Meinheld. This has been fixed so that
   :func:`~falcon.deprecated` no longer relies on the availability of
-  interpreter-specific stack frame instrospection capabilities. (`#1882 <https://github.com/falconry/falcon/issues/1882>`__)
+  interpreter-specific stack frame introspection capabilities. (`#1882 <https://github.com/falconry/falcon/issues/1882>`__)
```

Comment:
This is also a bit debatable, not the spelling itself, but the fact that we treat changelogs more or less as immutable. But just a technical or spelling fix is fine I guess as long as it doesn't change the meaning. :+1: